docs/rfcs/XeGPU.md

```diff
@@ -344,52 +368,76 @@ Within the sg_map, `wi_layout` specifies the layout of work items, describing th
      	into tensor_desc<8x16xbf16, #sg_map_a>
 ```
 
-wi_data_size refers to the data size mapped to indiviudal work item, and sg_map_size to the collective size by all the work items as specified by sg_map. distribute_unit_size represents the minimun size of 2D tensor to be distributed to work items in a subgroup. tensor_size refers to the size of the tensor sepcified by tensor_desc.
-In the example above, wi_data_size is 1, sg_map_size is 16, tensor_size is 128.
+With `sg_map` attribute attached to tensor_desc, xegpu.load_nd operates in SIMT flavor and returns back a fragement associated with individual work item. The tensor_desc in the first example below specifies a tensor of 8x16 elements, which is distributed 8 times so each work item gets <8x1xbf16>. The second example shows the each work gets <8x2xint8> with 2 int8 elements packed as one 16-bit data.  
```

Comment:
The second example shows the each work gets <8x2xint8> with 2 int8 elements packed as one 16-bit data. ->
The second example shows the each work **item** gets <8x2xint8> with 2 int8 elements packed as one 16-bit data.